web/app/authenticators/torii.ts

```diff
@@ -15,11 +15,17 @@ export default class ToriiAuthenticator extends Torii {
      * A rejecting promise indicates invalid session data and will result
      * in the session being invalidated or remaining unauthenticated.
      */
-    return this.fetchSvc.fetch("/api/v1/me", {
+    await this.fetchSvc.fetch("/api/v1/me", {
```

Comment:
So this needs to return a promise, and that promise, upon success, needs to return the data that will be set on the session service.  You can combine the two calls so it has the expected return value, and resolve value with something like this:

```
const data = await super.restore(...arguments);
    /**
     * A rejecting promise indicates invalid session data and will result
     * in the session being invalidated or remaining unauthenticated.
     */
    return this.fetchSvc.fetch("/api/v1/me", {      method: "HEAD",
      headers: {
        "Hermes-Google-Access-Token": data.access_token,
      },
    }).then(() => return data);

```

Reply:
Good call!